index.d.ts

```diff
@@ -32,6 +32,25 @@ export interface Options {
 	@default true
 	*/
 	readonly waitForChildren?: boolean;
+
+	/**
+ 	If defined, the predicate will be called for each element that matches the selector. If it returns `true`, the element will be returned.
+
+	For example, if the content is dynamic or a selector can't be specific enough - you can check the `.textContent` of each element and only match the one that has the required text.
+
+	@example
```

Comment:
The example is missing the JS part, like in your issue.

And this example also needs to be added to the readme.

Generally, I try to keep index.d.ts and readme in sync as much as possible.